rskj-core/src/main/java/co/rsk/RskContext.java

```diff
@@ -1595,12 +1595,13 @@ private Web3WebSocketServer getWeb3WebSocketServer() {
                             new BlockchainBranchComparator(getBlockStore())
                     )
             );
-            RskJsonRpcHandler jsonRpcHandler = new RskJsonRpcHandler(emitter, jsonRpcSerializer);
+            RskWebSocketJsonRpcHandler jsonRpcHandler = new RskWebSocketJsonRpcHandler(emitter, jsonRpcSerializer);
             web3WebSocketServer = new Web3WebSocketServer(
                     rskSystemProperties.rpcWebSocketBindAddress(),
                     rskSystemProperties.rpcWebSocketPort(),
                     jsonRpcHandler,
-                    getJsonRpcWeb3ServerHandler()
+                    getJsonRpcWeb3ServerHandler(),
+                    rskSystemProperties.rpcWebSocketServerWriteTimeout()
```

Comment:
I don't know if this is that useful, but I would re-arrange the parameters so we have the "values" first and then the objects. For example, the constructor could be `Web3WebSocketServer(bindAddress, port, timeout, handler, handler)`

Reply:
mmm this is kind of arbitrary, there are some cases when you want to have certain objects/values closer, and this particular case has no restrictions or relevant data to bring those params closer